torch/_tensor_docs.py

```diff
@@ -432,45 +432,9 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('all',
                r"""
-.. function:: all() -> bool
+all(dim=None, keepdim=False, out=None) -> Tensor
```

Comment:
*, out since the out argument is kwarg only (or if it's not it should be)

Reply:
Done!